### PR TITLE
rubberband: use libsamplerate

### PR DESCRIPTION
**Pull request by Andarwinux**

### CODE REVIEW

**Comment by NotTsunami** [comment]
This is one that I still hope lands, sometime.

**Comment by Andarwinux** [comment]
It seems shinchiro is not interested. In any case, users who want libsamplerate can build the mpv locally. Nowadays, with ccache, modern machines can build it in less than ten minutes.

**Comment by shinchiro** [comment]
I've very limited time nowadays to work on this project (plus low motivation). Testing new changes will take longer time. I guess this changes can be upstream earlier but it got mysteriously deleted

**Comment by Andarwinux** [comment]
Because rubberband or mpv's implementation of rubberband has a bug see https://github.com/mpv-player/mpv/issues/12905. libsamplerate or fftw don't help with this, and the hack workaround in #12905 only leads CUDA OOM(NVIDIA vulkan hwdec is broken) . So I don't use rubberband anymore. Also, rubberband AArch64 needs to be patched, but this was fixed just upstream.

**Comment by NotTsunami** [comment]
> Because rubberband or mpv's implementation of rubberband has a bug see [mpv-player/mpv#12905](https://github.com/mpv-player/mpv/issues/12905). libsamplerate or fftw don't help with this, and the hack workaround in #12905 only leads CUDA OOM(NVIDIA vulkan hwdec is broken) . So I don't use rubberband anymore. Also, rubberband AArch64 needs to be patched, but this was fixed just upstream.

Shame. No worries. Thanks again.

File: packages/libsamplerate.cmake [hunk 1]
```diff
@@ -0,0 +1,22 @@
+ExternalProject_Add(libsamplerate
+    GIT_REPOSITORY https://github.com/libsndfile/libsamplerate.git
+    SOURCE_DIR ${SOURCE_LOCATION}
+    GIT_CLONE_FLAGS "--filter=tree:0"
+    UPDATE_COMMAND ""
+    GIT_REMOTE_NAME origin
```
Comment (shinchiro) [inline packages/libsamplerate.cmake]: Unneeded

Comment (Andarwinux) [reply packages/libsamplerate.cmake]: done.

**Pull request merged**